pkgs/tools/misc/etcher/default.nix

```diff
@@ -0,0 +1,99 @@
+{ lib
+, stdenv
+, fetchurl
+, patchelf
```

Comment:
this `patchelf` argument is not used anywhere and could be removed

Reply:
Thanks, I've fixed that now